lc39.d.ts

```diff
@@ -44,8 +44,15 @@ export interface Options extends FastifyServerOptions {
   exposeMetrics?: boolean
   getMetrics?: (client: typeof prometheusClient) => Record<string, Metric>
   metricsOptions?: Partial<IMetricsPluginOptions>
+  customLevels?: Record<string, number>
 }
 
 declare function lc39(filePathOrServiceModule: string | FastifyPluginAsync, options?: Options): Promise<FastifyInstance>
 
+declare module 'fastify' {
+  export interface FastifyBaseLogger {
+    [level: string] : LogFn | LevelWithSilentOrString
```

Comment:
In this way, we could use every string here. It could be possible to relate with the custom levels option?

Reply:
The feature allows to create custom levels with custom names, so we have to allow every string to be present here. We are not replacing the current available levels of Pino.

What do you mean by relate to pino custom level options?

Btw I had to add `LevelWithSilentOrString` due to a field in the Pino BaseLogger interface: https://github.com/pinojs/pino/blob/master/pino.d.ts#L150C6-L150C45

And I don't think is possible to exclude literals from string type: https://stackoverflow.com/questions/51442157/type-for-every-possible-string-value-except

Reply:
Probably I misunderstood what you are proposing, and I don't think to know how to do it. Can You help me?